kratos/utilities/time_discretization.cpp

```diff
@@ -36,6 +38,13 @@ std::array<double, 2> BDF1::ComputeBDFCoefficients(const double DeltaTime) const
     return coefficients;
 }
 
+std::array<double, 2> BDF1::ComputeBDFCoefficients(const ProcessInfo& rProcessInfo) const
+{
+    KRATOS_ERROR_IF_NOT(rProcessInfo.Has(DELTA_TIME)) << "No DELTA_TIME "
```

Comment:
To economize on checks we could use the one in the process info (I think it is actually in the data value container).